### PR TITLE
Add Inazuma Eleven 3: Lightning Bolt, Bomb Blast and Team Ogre Attacks!

### PR DESCRIPTION
**Pull request by SeyaLeNeko**
Adds Inazuma Eleven 3: Lightning Bolt and Bomb Blast.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat [hunk 1]
```diff
@@ -0,0 +1,24 @@
+title = "Inazuma Eleven 3: Lightning Bolt and Bomb Blast"
+description = "Inazuma Eleven is a role-playing and sports video game for the Nintendo 3DS, developed and published by Level-5. It's the  the  third installment of the Inazuma Eleven games. There are two versions of the game, Lightning Bolt and Bomb Blast."
```
Comment (Leo626) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: Should start the sentence with `Inazuma Eleven 3` instead of just `Inazuma Eleven`.

Comment (SeyaLeNeko) [reply games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: Okay i will add the Team Orge Attacks thanks for the review

**Comment by SeyaLeNeko** [comment]
I didn't found the title ID in jpn release

File: games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat [hunk 2]
```diff
@@ -0,0 +1,34 @@
+title = "Inazuma Eleven 3: Lightning Bolt, Bomb Blast and Team Ogre Attacks!"
+description = "Inazuma Eleven 3 is a role-playing and sports video game for the Nintendo 3DS, developed and published by Level-5. It's the  the  third installment of the Inazuma Eleven games. There are three versions of the game, Lightning Bolt, Bomb Blast and Team Ogre Attacks!."
```
Comment (Leo626) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: After reading the Wikipedia page for this, I found out that the Japanese version is in a compilation so you should also add this add the end.

`All three versions of this game were included in an updated re-release compilation titled \"Inazuma Eleven 1-2-3: Endo Mamoru's Legend\".`

**Comment by Leo626** [comment]
The `jpn` release is in a compilation called Inazuma Eleven 1-2-3: Endo Mamoru's Legend and looking at the title IDs in 3dsdb, each `eur` version of the game has two title IDs making it seven title IDs in total including the `jpn` release.

The `release_date` is off for the first release and the third version has a different one. I would prefer you order the releases like this. The `jpn` release only has one title ID.

```
[[releases]]
title = "00040000000F7D00"
region = "eur"
release_date = "2013-09-27"

[[releases]]
title = "00040000000F7B00"
region = "eur"
release_date = "2013-09-27"

[[releases]]
title = "00040000000F7F00"
region = "eur"
release_date = "2014-02-14"

[[releases]]
title = "00040000000F7E00"
region = "eur"
release_date = "2013-09-27"

[[releases]]
title = "00040000000F7C00"
region = "eur"
release_date = "2013-09-27"

[[releases]]
title = "00040000000F8000"
region = "eur"
release_date = "2014-02-14"

[[releases]]
title = "00040000000BB800"
region = "jpn"
release_date = "2012-12-27"
```

**Comment by SeyaLeNeko** [comment]
Ok I'll do it and thanks for the infos

File: games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat [hunk 3]
```diff
@@ -0,0 +1,39 @@
+title = "Inazuma Eleven 3: Lightning Bolt, Bomb Blast and Team Ogre Attacks!"
+description = "Inazuma Eleven 3 is a role-playing and sports video game for the Nintendo 3DS, developed and published by Level-5. It's the  the  third installment of the Inazuma Eleven games. There are three versions of the game, Lightning Bolt, Bomb Blast and Team Ogre Attacks!.All three versions of this game were included in an updated re-release compilation titled \"Inazuma Eleven 1-2-3: Endo Mamoru's Legend\"."
```
Comment (Leo626) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: There should be a space before the last sentence.

**Comment by Leo626** [comment]
One last thing to do is to update the directory name to `inazuma-eleven-3-lightning-bolt-and-bomb-blast-and-team-ogre-attacks`. If you don't how to do it, then I can do it after I merge this.

**Comment by SeyaLeNeko** [comment]
Yeah I don't know, can you tell me how to do that?

File: games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat (hunk 3 above)
Comment (CodingKoopa) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: > It's the  the  third installment

There are two "the"s here.

Comment (CodingKoopa) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: > It's the  the  third installment

Remove the unnecessary spaces here.

Comment (CodingKoopa) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: > Lightning Bolt, Bomb Blast and Team Ogre Attacks!

Put each release of the game here in quotes.

**Comment by CodingKoopa** [comment]
> One last thing to do is to update the directory name to inazuma-eleven-3-lightning-bolt-and-bomb-blast-and-team-ogre-attacks. If you don't how to do it, then I can do it after I merge this.

I'm not sure if there should be an "and" after lightning bolt? It makes the internal title used for the wiki more confusing, but is more grammatically correct.

File: games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat [hunk 4]
```diff
@@ -0,0 +1,39 @@
+title = "Inazuma Eleven 3: Lightning Bolt, Bomb Blast and Team Ogre Attacks!"
+description = "Inazuma Eleven 3 is a role-playing and sports video game for the Nintendo 3DS, developed and published by Level-5.It's the third installment of the Inazuma Eleven games. There are three versions of the game, \"Lightning Bolt\", \"Bomb Blast\" and \"Team Ogre Attacks!\".All three versions of this game were included in an updated re-release compilation titled \"Inazuma Eleven 1-2-3: Endo Mamoru's Legend\"."
```
Comment (CodingKoopa) [inline games/inazuma-eleven-3-lightning-bolt-and-bomb-blast/game.dat]: > It's the third installment
I should have specified - you still need a space before "It's".

**Comment by SeyaLeNeko** [comment]
Is that okay?

**Comment by Leo626** [comment]
@CodingKoopa Oh yeah, you are right. The `and` after `lightning bolt` should be removed so the directory name should be `inazuma-eleven-3-lightning-bolt-bomb-blast-and-team-ogre-attacks`.

**Comment by SeyaLeNeko** [comment]
Ok but how can I change the directory name?

**Comment by CodingKoopa** [comment]
I don't think it can be done through the GitHub web interface.

**Comment by Leo626** [comment]
Github doesn't have that feature. It has to be done from a git client. If you're not sure how to do it, then OK.

**Comment by SeyaLeNeko** [comment]
Ok done

**Pull request merged**